Userland/Libraries/LibCore/IODevice.h

```diff
@@ -84,6 +86,8 @@ class IODevice : public Object {
     ByteBuffer read_all();
     String read_line(size_t max_size = 16384);
 
+    size_t copy_from(const IODevice& source, size_t buffer_size = 65536);
```

Comment:
This buffer size constant seems to be shared between the kernel and usermode, maybe it should go in a shared header?